cicecore/cicedynB/analysis/ice_history_shared.F90

```diff
@@ -667,7 +667,7 @@ subroutine construct_filename(ncfile,suffix,ns)
       character (len=1) :: cstream
       character(len=*), parameter :: subname = '(construct_filename)'
 
-        iyear = nyr + year_init - 1 ! set year_init=1 in ice_in to get iyear=nyr
+        iyear = nyr
```

Comment:
nyr is now the model year.  There is no longer this dependence on year_init.  year_init just defines the model start year and plays no other role.  I think this was one of the main things that needed to change to make it easier to use and more robust.  I preserved use of the variables, nyr, month, mday, sec.  We could change "nyr" to "year" and "mday" to "day" or something else if it makes things clearer.

Reply:
Since nyr has always (to me) meant 'number of the year' (i.e. a counter, POP convention again), could we use just 'year'?  Or is that used for something else?  iyear signalled the value of year as an integer quantity (although it might have been a real-values variable - I don't remember that kind of detail), e.g. 1996.  mday meant day of the month - I think that one's okay.